src/app/lorax/directives/modal-issue.js

```diff
@@ -77,7 +78,26 @@ define(['angular', 'jquery'], function (angular, $) {
             // in /src/app/lorax/directives/window.js (onBgModeChange) which in turn
             // sets the background color of the issue modal based on the status of
             // the current issue.
-            this._windowService.setBgMode(this._$scope.issue.getStatusDescription(), false);
+            this._windowService.setBgMode(this._$scope.modalIssue.issue.getStatusDescription(), false);
+
+            var figure = $('figure');
```

Comment:
Generally yes but, I know for a fact that there is only one figure element on the page but, with that said, to be more future proof, it is probably better to just give it an id or class.